packages/waku/src/lib/middleware/dev-server-impl.ts

```diff
@@ -27,6 +27,7 @@ import { rscManagedPlugin } from '../plugins/vite-plugin-rsc-managed.js';
 import { rscDelegatePlugin } from '../plugins/vite-plugin-rsc-delegate.js';
 import { mergeUserViteConfig } from '../utils/merge-vite-config.js';
 import type { ClonableModuleNode, Middleware } from './types.js';
+import { typegenPlugin } from '../plugins/vite-plugin-typegen.js';
```

Comment:
Let's name the file and the function so that it implies a feature for `fs-router` or managed mode (rscManagedPlugin).